test/DynamoCoreTests/PackageDependencyTests.cs

```diff
@@ -502,5 +503,22 @@ public void PackageDependencyStatechangeTestAfterLoadingPackage()
             Assert.AreEqual(new PackageDependencyInfo("Dynamo Samples", new Version("2.0.0")), lastPackage);
             Assert.AreEqual(PackageDependencyState.Loaded, lastPackage.State);
         }
+
+        [Test]
+        public void ResolveDummyNodesOnDownloadingPackage()
+        {
+            string path = Path.Combine(TestDirectory, @"core\packageDependencyTests\ResolveDummyNodesOnDownloadingPackage.dyn");
+            OpenModel(path);
+
+            var dummyNodes = CurrentDynamoModel.CurrentWorkspace.Nodes.OfType<DummyNode>();
+            Assert.AreEqual(2, dummyNodes.Count());
+
+            // Load the HowickMaker dll and check that the dummy nodes have been resolved.
+            string packageDirectory = Path.Combine(TestDirectory, @"core\packageDependencyTests\HowickMaker");
+            LoadPackage(packageDirectory);
+
+            dummyNodes = CurrentDynamoModel.CurrentWorkspace.Nodes.OfType<DummyNode>();
+            Assert.AreEqual(0, dummyNodes.Count());
```

Comment:
Can you also assert the value of some node in the graph which depends on the dummy node being resolved?

Reply:
Yes, asserting the value of the node in the test.